lib/pages/home.dart

```diff
@@ -1,170 +1,170 @@
+      monthlyContributions = model.contributions
+          .fold(<_ContributionTile>[],
+              (List<_ContributionTile> list, Contribution contib) {
+            String key =
+                '${contib.year.toString()}-${contib.month.toString().padLeft(2,'0')}';
+            _ContributionTile tile;
+            if (!list.any((c) => c.yearMonth == key)) {
+              tile =
+                  new _ContributionTile(year: contib.year, month: contib.month);
+              list.add(tile);
+            } else {
+              tile = list.firstWhere((c) => c.yearMonth == key);
+            }
+            tile.contributions.add(contib.value);
+            return list;
+          })
+          .reversed
+          .toList();
+    }
+
+    @override
+    Widget build(BuildContext context) {
+      return new Scaffold(
+          appBar: new AppBar(
+            title: new Text('GitHub Grass'),
+            actions: <Widget>[
+              new IconButton(
+                icon: new Icon(Icons.refresh),
+                onPressed: model.update,
+              )
+            ],
+          ),
+          body: new ListView.builder(
+            itemCount: monthlyContributions.length + 1,
+            itemBuilder: (BuildContext context, int index) {
+              if (index == 0) {
+                return _createIdentityWidget();
+              } else {
+                _ContributionTile monthlyContrib =
+                    monthlyContributions[index - 1];
+                return _createMonthlyContributionMap(monthlyContrib);
+              }
+            },
+          ));
+    }
+
+    Widget _createIdentityWidget() {
+      DateTime today = new DateTime.now();
+
+      Contribution c = model.contributions.firstWhere((c) =>
+          c.year == today.year && c.month == today.month && c.day == today.day);
+      int value = c.value ?? 0;
+
+      String todayLabel =
+          '${today.year.toString()}-${today.month.toString().padLeft(2,'0')}-${today.day.toString().padLeft(2,'0')}';
+
+      return new Padding(
+        padding: const EdgeInsets.symmetric(vertical: 10.0),
+        child: new Center(
+          child: new Column(
+            children: <Widget>[
+              new CircleAvatar(
+                backgroundImage: new NetworkImage(
+                    'https://avatars.githubusercontent.com/${model.username}'),
+              ),
+              new Text(
+                model.username,
                 style: new TextStyle(
-                  fontSize: 10.0,
+                  fontSize: 20.0,
                 ),
-              );
-              valueText = _createValueBox(c);
-              return new Flexible(
-                flex: 1,
-                child: monthlyContrib.isPresentMonth()
-                    ? valueText
-                    : new Center(
-                        child: valueText,
+              ),
+              value > 0
+                  ? new Text(
+                      'Today\'s contribution is ${value}! 👍',
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Avoid using braces in interpolation when not needed.